Split specPath to pass separate arguments to provider command

raphink opened the pull request.

izgeri commented:
Hey @raphink - thanks for submitting this! Is it connected to an issue? If not, can you please file one that gives a clearer motivation for this change?

Before this change can be merged, we'll also need you to sign off on your commit. You can find more info and instructions for adding a sign off [here](https://github.com/cyberark/community/blob/master/Conjur/CONTRIBUTING.md#when-the-repo-does-not-include-the-cla). Please let us know if you have any questions!

raphink commented:
Thanks for the comments @izgeri and @sgnn7! I've added an issue (#157). In order to not break compatibility, maybe a flag could be used?

```yaml
FOO: !var:split /path/to/key subkey
```

sgnn7 commented:
Blocked on https://github.com/cyberark/summon/issues/157#issuecomment-654351691
